packages/react-dom/src/__tests__/ReactDOMFizzStatic-test.js

```diff
@@ -430,6 +434,7 @@ describe('ReactDOMFizzStatic', () => {
       result.prelude.pipe(writable);
     });
     expect(getVisibleChildren(container)).toEqual(undefined);
+    expect(errors).toEqual([]);
```

Comment:
The test used to fail due to `prerenderToNodeStream` not existing. Since that API exists everywhere now, the only error happens in `onError` calling `console.error(postponedValue)`. However, we ignore `console.error(someErrorInstance)` in our testing infra and [it would take a while to update all the tests to also assert on prod errors](https://github.com/facebook/react/pull/34634).

In dev the test failed because also called `console.error` with the environment name